src/client/components/immutable-dropdown/immutable-dropdown.tsx

```diff
@@ -34,7 +34,7 @@ export interface ImmutableDropdownProps<T> extends React.Props<any> {
   equal: (a: T, b: T) => boolean;
   renderItem: (a: T) => string;
   keyItem: (a: T) => any;
-  onChange: (newItem: T, isValid: boolean, path: string) => void;
+  onChange: (newItem: any, isValid: boolean, path: string) => void;
```

Comment:
I don't need it, but it doesn't make sense before the change. The newItem is not a new list item, it's an "instance". So nothing to do with `<T>`.